pkg/ccl/importccl/read_import_proc.go

```diff
@@ -619,15 +620,21 @@ func makeRowErr(file string, row int64, format string, args ...interface{}) erro
 // ingestKvs drains kvs from the channel until it closes, ingesting them using
 // the BulkAdder. It handles the required buffering/sorting/etc.
 func ingestKvs(ctx context.Context, adder storagebase.BulkAdder, kvCh <-chan kvBatch) error {
-	// TODO(dt): allow configuring these.
-	const sortBatchSize = 1000000
-	// TODO(dt): track byte size as well as key count to trigger flushes.
+	const sortBatchSize = 48 << 20 // 48MB
```

Comment:
I think we also need to track total across buckets and have a hard flush limit, otherwise imports with more tables or tables with more indexes can use unbounded buffering memory.